main.go

```diff
@@ -15,10 +15,11 @@ import (
 )
 
 var (
-	queueLength    = flag.Int("queue-length", 100, "Length of the queue of workers")
-	port           = flag.String("port", "8080", "Server port")
+	queueLength = flag.Int("queue-length", 100, "Length of the queue of workers")
+	port        = flag.String("port", "8080", "Server port")
```

Comment:
Please run `go fmt` before submitting or use a go linter (that would run the command for you).